### DISCUSSION TITLE
Multiple Nuxt Apps in same repo.

### DISCUSSION THREAD

**Discussion by gilesbutler** [nuxt/nuxt#16535] [Questions]
Hey,

Congrats on the Nuxt 3 release, awesome work!!

I was wondering if it's possible to setup multiple Nuxt 3 apps in the same repo so they can share components, content etc.

We'd like to have our marketing pages and admin site in the same repo as they share a lot of the same code. But they live on different subdomains so if one goes offline the other is still ok.

I was look at the following setup https://github.com/nuxt/nuxt.js/issues/1785#issuecomment-334415666 but it doesn't seem to work with Nuxt 3 as the `--config-file` argument doesn't look like it's working with Nuxt 3 yet.

I think I saw in one of the original Nuxt 3 announcements a line about support for multiple apps but I haven't been able to find any other info on this since.

Keen to hear any thoughts on how to set this up.

Thanks
Giles

**Comment by TheAlexLichter** [comment]
This is possible now via `extends`. See https://github.com/nuxt/framework/issues/3222
